api/broadcast/data.go

```diff
@@ -0,0 +1,57 @@
+package broadcast
+
+import (
+	"encoding/json"
+	"net/http"
+	"github.com/gorilla/mux"
+	"fmt"
+	"bytes"
+	"io/ioutil"
+	"github.com/ShyftNetwork/blockexplorer_api/logger"
+)
+
+// Broadcast broadcasts tx
+func Broadcast(w http.ResponseWriter, r *http.Request) {
+	// Example return result (returns tx hash):
```

Comment:
@dutterbutter do you need this for later?

Reply:
@priom I believe so yes, Tim wrote it and told me it is for the anti-censorship mechanism